zendserver/init.sls

```diff
@@ -136,6 +136,20 @@ alternative-phpize:
     - group: adm
     - mode: 740
 
+/etc/zendserver/changeJobQueueRetryCount.php:
+  file.managed:
+    - source: salt://zendserver/files/changeJobQueueRetryCount.php
+    - user: root
+    - group: adm
+    - mode: 740
```

Comment:
This is a file, so mode should not start with a 7 right?

Reply:
It's a `+x` mode, making the file executable (rwx r-- ---), the same as for `changePhpVersion.php`, because both files have shebang lines `#!/usr/bin/env php` and are treated like cli executables which accept arguments from `$argv[]`

If you want to use them as 640, we need to change the way how it launches from
```
zendserver.changePhpVersion:
 cmd.run:
   - name: /etc/zendserver/changePhpVersion.php admin {{ zend_api_key }} {{ php_version }}
```
to something like
```
zendserver.changePhpVersion:
 cmd.run:
   - name: sudo /usr/bin/php /etc/zendserver/changePhpVersion.php admin {{ zend_api_key }} {{ php_version }}
```